classquiz/routers/login.py

```diff
@@ -102,19 +102,27 @@ def verify_webauthn(data, fidocredentialss: list[FidoCredentials], login_session
 @router.post("/start")
 async def start_login(data: StartLoginInput):
     user = (
+        # TODO Find out why the fidocredentialls call fails
         await User.objects.select_related("fidocredentialss")
```

Comment:
Maybe try `fidocredentials` instead of `fidocredentialss`, so one `s` less?

Reply:
Sorry, I should have removed this TODO. In fact, this fidocredentialls error was resolved by updating the relevant class in db/models.py to have the proper ormar field definitions. It is TODONE!